python/lsst/analysis/tools/atools/metadataMetrics.py

```diff
@@ -64,8 +64,10 @@ def validate(self):
 class DatasetMetadataMetricTool(MetadataMetricTool):
     """Tool designed to extract values from metadata of data products"""
 
-    metricsStoredAsDict = DictField[str, bool](
```

Comment:
As this is a changed config, does this need to be deprecated?

Reply:
I just introduced this config last week for a specific ticket. I did a quick search, and no one else is using it (even if they were, it wouldn’t have worked for the reasons discussed in this ticket). So, I think it’s fine to remove it without deprecation.